libraries/provision/install_sync_gateway.py

```diff
@@ -23,24 +23,6 @@ def __init__(self,
 
         self._version_number = version_number
         self._build_number = build_number
-        self._valid_versions = [
-            "1.1.0",
-            "1.1.1",
-            "1.2.0",
-            "1.2.1",
-            "1.3.0",
-            "1.3.1",
-            "1.3.1.2",
-            "1.4.0",
-            "1.4",
-            "1.4.0.1",
-            "1.4.0.2",
-            "1.4.1",
-            "1.4.1.1",
-            "1.4.2",
-            "1.5.0"
-        ]
```

Comment:
reason for removing self._valid_versions?

Reply:
We are not using it anymore. We don't validate the version before installing, we do it after. We used to do it, not anymore.